docs/docs-content/clusters/data-center/vmware/permissions.md

```diff
@@ -7,186 +7,227 @@ sidebar_position: 60
 
 </TabItem>
 
 </Tabs>
+
+### Spectro Role Assignments
+
+The privileges associated with the _Spectro role_ must be granted via role assignments on specific vSphere objects for
+either the user or a group containing the user. Review the required role assignments to ensure that your user has all
+required privileges on all required objects.
+
+| **vSphere Object**                           | **Propagation** | **Role**       | **Condition**                                                                                                                             |
+| -------------------------------------------- | --------------- | -------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| **Target Network**                           | Yes             | _Spectro role_ |                                                                                                                                           |
+| **Target Datastore**                         | Yes             | _Spectro role_ |                                                                                                                                           |
+| **All ESXi hosts within the Target Cluster** | No              | _Spectro role_ |                                                                                                                                           |
+| **spectro-templates Folder**                 | Yes             | _Spectro role_ | For airgap installs, it must be manually created in advance and permissions assigned. For connected installs it is created automatically. |
```

Comment:
@karl-cardenas-coding I'd like to confirm this. I'm not aware of any automated mechanism by which this folder is created. I kinda think it's a manual process in all scenarios. But I'm probably wrong 🤷🏼‍♂️

Reply:
You're right @TylerGillson